.github/workflows/review_pull_request.yml

```diff
@@ -34,15 +34,15 @@ jobs:
       # template-xml
 
       - name: Run Detekt on template-xml
-        working-directory: ./template-xml
+        working-directory: ./deprecated/template-xml
         run: ./gradlew detekt
 
       - name: Run Android Lint on template-xml
-        working-directory: ./template-xml
+        working-directory: ./deprecated/template-xml
         run: ./gradlew lint
 
       - name: Run unit tests with Kover on template-xml
-        working-directory: ./template-xml
+        working-directory: ./deprecated/template-xml
         run: ./gradlew koverXmlReport
```

Comment:
I think we should remove all workflows for deprecated projects. We should NOT waste our resources to verify somethings it's not being developed or maintained cc @nimblehq/chapter-android  🙏